src/main/java/appeng/client/render/cablebus/CableBusRenderState.java

```diff
@@ -154,6 +156,11 @@ public void setCableBusAdjacent( EnumSet<EnumFacing> cableBusAdjacent )
 		return this.boundingBoxes;
 	}
 
+	public EnumMap<EnumFacing, Long> getPartFlags()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "Map" rather than the implementation "EnumMap". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1319)